descriptor/descriptor.go

```diff
@@ -42,8 +42,8 @@ import (
 	"fmt"
 	"io/ioutil"
 
+	protobuf "github.com/golang/protobuf/descriptor"
```

Comment:
Should this be: `protobuf "github.com/golang/protobuf/protoc-gen-go/descriptor"`?

Same change in descriptor_test.go